frontend/src/scenes/onboarding/sdks/SDKs.tsx

```diff
@@ -66,6 +67,14 @@ export function SDKs({
         }
     }, 2000)
 
+    const HelpCard = (): JSX.Element => (
```

Comment:
Nit: `HelpCard` isn't very descriptive, something along the lines of `InviteTeamMemberCard` might be better